### DISCUSSION TITLE
Unable to set a github package as public, in a public repo (says locked by org settings)

### DISCUSSION THREAD

**Discussion by PicoCreator** [community/community#65931] [Packages]
### Select Topic Area

Bug

### Body

I been trying to configure some docker build images to be public, in a public repo here:

https://github.com/orgs/RWKV/packages/container/rwkv-infctx-trainer/settings

However it shows the following message blocking the process.

<img width="717" alt="Screenshot 2023-09-04 at 9 57 35 AM" src="https://github.com/community/community/assets/17175484/edc25ec4-4da1-49d1-baca-e731cf1d42f7">

We have tried using both the RWKV organization admin, and the repo admin to change the visibility level of the package, but it still says it is disabled.

We are unsure on how to proceed forward

**Comment by GabrieleMajocchi** [comment]
It seems like you're encountering an issue with changing the visibility level of a Docker container image in a GitHub repository. To make a Docker container image public on GitHub Packages, you typically need to follow these steps:

-Repository Ownership: Make sure that you have the necessary permissions to change the visibility settings of the repository. You should either be the organization admin or the repository admin.

-Repository Settings: Go to the repository where you have the Docker container image hosted.

-Packages Settings: Navigate to the "Packages" settings for the repository. In your case, you provided the following URL: https://github.com/orgs/RWKV/packages/container/rwkv-infctx-trainer/settings.

-Change Visibility: Look for an option to change the visibility level of the package. It should typically be something like "Visibility" or "Access Control."

-Select Public: Choose the "Public" option to make the Docker container image public.

-Save Changes: Be sure to save your changes.

If you're still encountering issues after following these steps, here are some additional troubleshooting tips:

Ensure that there are no specific repository settings, permissions, or restrictions that might be preventing you from changing the visibility.

Double-check if there are any GitHub Actions workflows or scripts in your repository that might be interfering with the visibility settings.

If you're using GitHub Actions to build and publish the Docker image, make sure your workflow is configured correctly to publish the image as public.

Check if you have any GitHub Actions secrets or environment variables that might be affecting the visibility settings.

Review GitHub's documentation and any relevant GitHub issues or discussions for known issues related to changing package visibility.

If you've tried these steps and are still having trouble, you may want to reach out to GitHub support for further assistance, as it could be a specific issue related to your repository or organization settings that requires their attention.

**Reply by jvyden** [reply]
Posting a question into ChatGPT isn't a good form of help...

**Reply by ghostinhershell** [reply]
Hi @GabrieleMajocchi, 

We’ve clarified our stance on using generative AI tools like ChatGPT within our Community via [this announcement](https://github.com/orgs/community/discussions/69006). Please review the guidelines to ensure your post meets them as failure to adhere to those rules can result in action taken by our moderator team.  You can read our updated [Code of Conduct](https://github.com/community/community/blob/baa5cd9020ef3f2bbf86dca1191d9c90f9b0822d/CODE_OF_CONDUCT.md#reasonable-use-of-ai-generated-content) and the [announcement](https://github.com/orgs/community/discussions/69006) for more details. Thank you for helping us maintain an authentic and beneficial space for everyone.

**Comment by jvyden** [comment]
I found the solution to this - turns out the setting is in the organization settings, not the repository settings.

![image](https://github.com/community/community/assets/40577357/ea071a16-2886-4f94-a556-9bbad126fad7)

**Reply by Erudition** [reply]
Ugh! Thank you. It was in my user settings, no org. Although that second section is all I see.

All these places talking about a nonexistent "Repo Settings -> Packages" screen....

**Reply by deavmi** [reply]
Thank you! GitHub has ass UX.
